molecule/default/playbook.yml

```diff
@@ -11,3 +11,4 @@
     - role: sb-debian-base
       vars:
         prebootstrap: true
+        swapfile: false
```

Comment:
@speedlight what is the purpose behind this flag?

Reply:
Sorry non related to this, was testing enabling bootstrap and swapfile :)